python/cog/director/director.py

```diff
@@ -301,6 +319,13 @@ def _handle_message(self, message_id, message_json) -> None:
             log.info("prediction succeeded")
             self._record_success()
 
+        span.set_attribute("prediction.status", tracker.status)
+        if tracker._response.completed_at:
+            completed_at_rfc3999_ms = tracker._response.completed_at.strftime(
```

Comment:
Why is this datestamp handled differently the the `started_at` and `created_at` datestamps?

Reply:
Yeah no good reason... I need to see if OpenTelemetry in Python by default emits millisecond precision and then I'll fix it all up